draft-ietf-quic-multipath.md

```diff
@@ -885,6 +884,33 @@ two or more active paths during the connection lifetime. Different applications
 Once the implementation has decided which paths to keep alive, it can do so by sending Ping frames
 on each of these paths before the idle timeout expires.
 
+## Migration
+
```

Comment:
Migration in RFC 9000 means "move from sending and receiving on path 1 to sending and receiving on path 2". I think that multipath implementation just replaces the RFC 9000 concept to "migration", replacing it by a multiphase process in which path 2 is explicitly tried and validated, and then path 1 might be kept going, or might be explicitly abandoned if not useful any more.

We are discussing here two exceptions to that process: NAT Rebinding, and CID renewal. Mabe we should make that into two sections, instead of a single migration section.

Reply:
I kept it in one section but am not talking about migration anymore but CID changes and NAT rebinding instead. Better?